src/components/EmailNotifications/VerifyEmailAddress.js

```diff
@@ -25,11 +26,18 @@ const VerifyEmailAddress = React.memo(function VerifyEmailAddress({
   const theme = useTheme()
   const [emailAddress, setEmailAddress] = useState('')
   const [emailInvalid, setEmailInvalid] = useState(false)
+  const [errorMessage, setErrorMessage] = useState('')
   const [termsAccepted, setTermsAccepted] = useState(false)
 
+  const [insideModal] = useInside('NotificationsModal')
+
   const handleEmailAddressBlur = useCallback(e => {
     const email = e.target.value
-    setEmailInvalid(!validateEmail(email))
+    const emailInvalid = !validateEmail(email)
+    setEmailInvalid(emailInvalid)
+    if (emailInvalid) {
+      setErrorMessage('Please enter a valid email address.')
+    }
   }, [])
 
   const handleEmailAddressChange = useCallback(e => {
```

Comment:
If we don't consolidate the state, we should probably also remove the error when changing the email input?